### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by shirayu**
- Related to #483

### CODE REVIEW

File: scripts/convert_original_stable_diffusion_to_diffusers.py [hunk 1]
```diff
@@ -276,7 +276,7 @@ def create_diffusers_schedular(original_config):
 
 
 def create_ldm_bert_config(original_config):
-    bert_params = original_config.model.parms.cond_stage_config.params
+    bert_params = original_config.model.params.cond_stage_config.params
```
[the remark this answers is not in the document]
Comment (shirayu) [reply scripts/convert_original_stable_diffusion_to_diffusers.py]: Thank you for your comment!
I fixed them and added an exception to `_typos.toml`.

### Note
After https://github.com/crate-ci/typos/issues/316 has been resolved, only the line should be is ignored.

File: _typos.toml [hunk 2]
```diff
@@ -4,8 +4,9 @@
 [default.extend-identifiers]
 
 [default.extend-words]
-NIN_="NIN" # NIN is used in scripts/convert_ncsnpp_original_checkpoint_to_diffusers.py
+NIN="NIN" # NIN is used in scripts/convert_ncsnpp_original_checkpoint_to_diffusers.py
 nd="np" # nd may be np (numpy)
+parms="parms" # parms is used in scripts/convert_original_stable_diffusion_to_diffusers.py
```
Comment (patrickvonplaten) [inline _typos.toml:9]:
```suggestion
params="params" # params is used in scripts/convert_original_stable_diffusion_to_diffusers.py
```

File: scripts/convert_original_stable_diffusion_to_diffusers.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (patrickvonplaten) [reply scripts/convert_original_stable_diffusion_to_diffusers.py]: Ah sorry - I see this is a typo in the original repo

**Review by patrickvonplaten — APPROVED** [review]
Good for me!

**Review by anton-l — APPROVED** [review]

**Pull request merged**